docs: add Bing Ads Google sign-in troubleshooting note (#18618)
PostHog/posthog.com: contents/docs/cdp/sources/bing-ads.md

## Supported tables

<SourceTables />

## Troubleshooting

> ### Can't connect an account that signs in with Google

Author: ```suggestion
### Can't connect if you sign in with Google
```

> PostHog authenticates through the Microsoft Advertising API, which only accepts a Microsoft account. If you sign in to Microsoft Advertising with a **Google account**, the connection can't complete — this is a Microsoft limitation, not specific to PostHog.

Author: ```suggestion
PostHog authenticates through the Microsoft Advertising API, which only accepts a Microsoft account. If you sign in to Microsoft Advertising with a **Google account**, the connection can't complete. This is a Microsoft limitation, not specific to PostHog.
```
